LibraBFT/Impl/Consensus/SafetyRules/SafetyRules.agda

```diff
@@ -96,8 +96,8 @@ verifyAndUpdatePreferredRoundM quorumCert safetyData = do
 verifyAuthorM : Maybe Author → LBFT (Either ErrLog Unit)
 verifyAuthorM author = do
   vs ← use (lSafetyRules ∙ srValidatorSigner)
-  maybeS vs (bail fakeErr) {-(ErrL (here' ["srValidatorSigner", "Nothing"]))-} $ λ validatorSigner →
-    maybeS
+  maybeS-RWST vs (bail fakeErr) {-(ErrL (here' ["srValidatorSigner", "Nothing"]))-} $ λ validatorSigner →
+    maybeS-RWST
```

Comment:
For comparison, this is an example where `maybeS-RWST` _is_ preferred, since the scrutinee is `author` (an argument to the function).